rebasehelper/specfile.py

```diff
@@ -1402,6 +1402,8 @@ def update_setup_dirname(self, dirname):
                     macros = [m for m in MacroHelper.filter(self.macros, level=-3) if m['name'] in ('name', 'version')]
                     # add all macros from spec file scope
                     macros.extend(MacroHelper.filter(self.macros, level=0))
+                    # avoid false-positives caused by replacing one character by a macro
```

Comment:
I really don't feel like false-positive is a suitable term here. How about simple `# omit extremely short macros`?